src/lib/Libwin/win_remote_shell.c

```diff
@@ -803,19 +830,24 @@ remote_shell_command(char *remote_host, char *pipename_append, BOOL connect_stdi
 
 	/* Set console title */
 	sprintf(console_title, "Connecting to %s...", remote_host);
-	SetConsoleTitle(console_title);
+	if (!SetConsoleTitle(console_title)) {
+		fprintf(stderr, "SetConsoleTitle failed on %s\n", remote_host);
```

Comment:
Probably log_err is better here

Reply:
This API is being called in remote_shell_command() function and the rest of the error logging is done using fprintf. I am not sure why but my guess would be because we are executing commands on shell and we want to display the error there.

Reply:
But if we use log_err then also it will be handled right? Do you have any logs where we can see it is logging in terminal, it will be helpful to understand.
Because I didn't understand, while creating the process, even if we get any error and that logged in that process shell, how will we know about it (if we don't log) ?